File: go/examples/cells/cells.go
package main

import (
	"os"

	uimd "uimd"
)

func main() {
	code := uimd.RunGeneratedAppMain(func() int {
		app := NewCellsUI()
		return uimd.RunGeneratedWindow(app, app.RuntimeOptions(), os.Args)
	})
	os.Exit(code)
}

File: java/examples/cells/Cells.java
import uimd.GeneratedWindowRuntime;

public class Cells extends CellsUI
{
    public static void main(String[] arguments)
    {
        int exitCode = GeneratedWindowRuntime.runGeneratedAppMain(() ->
        {
            Cells application = new Cells();
            return GeneratedWindowRuntime.runGeneratedWindow(
                application,
                application.runtimeOptions(),
                arguments);
        });
        System.exit(exitCode);
    }
}
